script/upgrades/MU03/MU03Checks.sol

```diff
@@ -0,0 +1,687 @@
+      }
+    }
+    console2.log("\tTrading limits set for all exchanges 🔒");
+  }
+
+  function verifyReserveFraction() public view {
+    address[] memory exchangesV1 = Arrays.addresses(
+      contracts.celoRegistry("Exchange"),
+      contracts.celoRegistry("ExchangeBRL"),
+      contracts.celoRegistry("ExchangeEUR")
+    );
+    uint256[] memory reserveFractions = Arrays.uints(2e22, 5e21, 5e21);
+    for (uint256 i = 0; i < exchangesV1.length; i++) {
+      if (Exchange(exchangesV1[i]).reserveFraction() != (reserveFractions[i] / 2)) {
+        console2.log("Reserve fraction not scaled down to correct value for exchange %s", exchangesV1[i]);
+        revert("Reserve fraction not scaled down correctly for all exchanges");
+      }
+    }
+    console2.log("\tReserve fraction scaled down correctly for all exchanges 🧾");
+  }
+
+  /* ================================================================ */
+  /* ======================== Circuit Breaker ======================= */
+  /* ================================================================ */
+
+  function verifyCircuitBreaker() public view {
+    console2.log("\n== Checking circuit breaker... ==");
+    verifyBreakerBox();
+    verifyMedianDeltaBreaker();
+    verifyValueDeltaBreaker();
+  }
+
+  function verifyBreakerBox() public view {
+    // verify that breakers were set with trading mode 3
+    if (
+      breakerBox.breakerTradingMode(medianDeltaBreaker) != 3 || breakerBox.breakerTradingMode(valueDeltaBreaker) != 3
+    ) {
+      console2.log("Breakers were not set with trading halted ❌");
+      revert("Breakers were not set with trading halted");
+    }
+    console2.log("\tBreakers set with trading mode 3");
+
+    // verify that rate feed dependencies were configured correctly
+    address cEurDependency = breakerBox.rateFeedDependencies(cEURUSDCConfig.referenceRateFeedID, 0);
+    address cBrlDependency = breakerBox.rateFeedDependencies(cBRLUSDCConfig.referenceRateFeedID, 0);
+    require(cEurDependency == cUSDUSDCConfig.referenceRateFeedID, "cEUR/USDC dependency not set correctly");
+    require(cBrlDependency == cUSDUSDCConfig.referenceRateFeedID, "cBRL/USDC dependency not set correctly");
+    console2.log("\tRate feed dependencies configured correctly 🗳️");
+
+    // verify that MedianDeltaBreaker && ValueDeltaBreaker were enabled for rateFeeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isMedianDeltaBreakerEnabled) {
+        (, , bool medianDeltaStatus) = breakerBox.rateFeedBreakerStatus(
+          poolConfigs[i].referenceRateFeedID,
+          medianDeltaBreaker
+        );
+        if (!medianDeltaStatus) {
+          console2.log("MedianDeltaBreaker not enabled for rate feed %s", poolConfigs[i].referenceRateFeedID);
+          revert("MedianDeltaBreaker not enabled for all rate feeds");
+        }
+
+        if (poolConfigs[i].isValueDeltaBreakerEnabled) {
+          (, , bool valueDeltaStatus) = breakerBox.rateFeedBreakerStatus(
```

Comment:
nit: there's a `breakerBox.isBreakerEnabled` that can be used as well

Reply:
👍🏼